src/map/unit.cpp

```diff
@@ -3093,6 +3096,11 @@ int unit_remove_map_(struct block_list *bl, clr_type clrtype, const char* file,
 			}
 			break;
 		}
+		case BL_NPC: {
```

Comment:
You don't have to wrap this in brackets since you aren't defining any variables.